Assets/Scripts/MenuButtons.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuButtons : MonoBehaviour {
+
+    int currentScene;
+
+	// Use this for initialization
+	void Start () {
+        currentScene = SceneManager.GetActiveScene().buildIndex;
+	}
+	
+    public void PressStartButton()
+    {
+        SceneManager.LoadScene(currentScene + 1);
+    }
+
+    public void StartLevelOne()
+    {
+        SceneManager.LoadScene(1);
+    }
+
+    public void Credits()
+    {
+        SceneManager.LoadScene("Credits");
+    }
+
+    public void ExitCredits()
+    {
+        SceneManager.LoadScene("MainMenu");
+    }
+
+    public void Quit()
+    {
+        //TODO: Debug.Log("Quit Program Input Accepted")
```

Comment:
still todo?

Reply:
I'm using it to make sure that I don't forget to remove the debug statement that let's me test the quit/exit button in the editor, once I finalized the art and finished the implementation of the main menu I was going to remove it